examples/wearable/UIComponents/contents/controls/indicator/index.html

```diff
@@ -17,7 +17,7 @@
 </head>
 
 <body>
-	<div class="ui-page">
+	<div class="ui-page" data-enable-go-to-top-button="true">
```

Comment:
May be considered short name of this option, eg. `data-go-to-top-button` ?

Reply:
Ok I shorten the name hopefully everywhere.